### PR TITLE
Update portal-overview.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3828
Added note about devices with scaling issues.

### CODE REVIEW

**Comment by e0i** [comment]
@mjcaparas A PR that introduces a clarifying note to address issue #3828 is available for review. The PR has been submitted by a writer from the freelance technical writing team.

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/portal-overview.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ Area | Description
 **(2) Main portal** | Main area where you will see the different views such as the Dashboards, Alerts queue, and Machines list.
 **(3) Community center, Localization,  Help and support, Feedback** | **Community center** -Access the Community center to learn, collaborate, and share experiences about the product. </br></br>  **Time settings** - Gives you access to the configuration settings where you can set time zones and view license information. </br></br>  **Help and support** - Gives you access to the Microsoft Defender ATP guide, Microsoft support, and Premier support.</br></br> **Feedback** - Access the feedback button to provide comments about the portal. 
 
+> [!NOTE]
+> For devices with high resolution DPI scaling issues, please see here for possible solutions - [Windows scaling issues for high-DPI devices](https://support.microsoft.com/help/3025083/windows-scaling-issues-for-high-dpi-devices)
```
Comment (nenonix) [inline windows/security/threat-protection/microsoft-defender-atp/portal-overview.md]:
```suggestion
> For devices with high resolution DPI scaling issues, please see [Windows scaling issues for high-DPI devices](https://support.microsoft.com/help/3025083/windows-scaling-issues-for-high-dpi-devices) for possible solutions.
```

**Comment by e0i** [comment]
@mjcaparas Any updates with the review? We'd like to follow-up. Thanks.

**Pull request merged**